<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>90</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="m_generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QFormLayout" name="formLayout_6">
         <item row="7" column="0">
          <widget class="QCheckBox" name="m_humanCanPlayAfterTimeoutCheck">
           <property name="text">
            <string>Human player can play after timeout</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="m_playersSidesOnClocksCheck">
           <property name="text">
            <string>Display players' sides on clocks </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="m_closeUnusedInitialTabCheck">
           <property name="text">
            <string>Close initial game tab if unused</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="m_highlightLegalMovesCheck">
           <property name="text">
            <string>Highlight legal moves</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="m_useFullUserNameCheck">
           <property name="text">
            <string>Use full name of human player</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="m_autoFlipBoardForHumanGamesCheck">
           <property name="text">
            <string>Auto flip board for human vs human</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="horizontalSpacing">
          <number>9</number>
         </property>
         <property name="verticalSpacing">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>13</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_siteEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="m_siteLabel">
           <property name="text">
            <string>PGN Site:</string>
           </property>
           <property name="buddy">
            <cstring>m_siteEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="m_defaultPgnOutFileLabel">
           <property name="text">
            <string>PGN output for single games:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="m_defaultPgnOutFileEdit">
             <property name="text">
              <string>games.pgn</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_defaultPgnOutFileBtn">
             <property name="text">
              <string>Browse...</string>
             </property>
             <property name="shortcut">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="m_tbPathLabel">
           <property name="text">
            <string>Syzygy tablebases path:</string>
           </property>
           <property name="buddy">
            <cstring>m_tbPathEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="m_tbPathEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="m_browseTbPathBtn">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_enginesTab">
      <attribute name="title">
       <string>Engines</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="EngineManagementWidget" name="m_engineManagementWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_gamesTab">
      <attribute name="title">
       <string>Games</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="GameSettingsWidget" name="m_gameSettings" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tournamentsTab">
      <attribute name="title">
       <string>Tournaments</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="TournamentSettingsWidget" name="m_tournamentSettings" native="true"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Games</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="m_concurrencyLabel">
              <property name="text">
               <string>Concurrent games:</string>
              </property>
              <property name="buddy">
               <cstring>m_concurrencySpin</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="m_concurrencySpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="m_tournamentDefaultPgnOutFileLabel">
              <property name="text">
               <string>PGN output:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="m_tournamentDefaultPgnOutFileEdit">
                <property name="text">
                 <string>tournaments.pgn</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_tournamentDefaultPgnOutFileBtn">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="m_tournamentDefaultEpdOutFileLabel">
              <property name="text">
               <string>EPD output:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="m_tournamentDefaultEpdOutFileEdit">
                <property name="text">
                 <string>tournaments.epd</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_tournamentDefaultEpdOutFileBtn">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameSettingsWidget</class>
   <extends>QWidget</extends>
   <header>src/gamesettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TournamentSettingsWidget</class>
   <extends>QWidget</extends>
   <header>src/tournamentsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EngineManagementWidget</class>
   <extends>QWidget</extends>
   <header>src/enginemanagementwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
